src/components/OrganizationScreen/OrganizationScreen.tsx

```diff
@@ -85,16 +108,62 @@ const organizationScreen = (): JSX.Element => {
       >
         <div className="d-flex justify-content-between align-items-center">
           <div style={{ flex: 1 }}>
-            <h2>{t('title')}</h2>
+            <h1>{t('title')}</h1>
           </div>
+          <Dropdown as={ButtonGroup} variant="none" data-testid="togDrop">
```

Comment:
Make this a separate component so that it can be used across various screens, Also make sure to handle the case where the user's name is too long

Reply:
added condition for long names
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/90516956/27eeb18b-e6de-4fee-a7e3-257d3779e3cd)